src/setupTests.js

```diff
@@ -1,9 +1,17 @@
 import { configure } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
+
 import '@testing-library/jest-dom';
+import { server } from './mocks/servers';
+
+beforeAll(() => {
+  server.listen();
+});
+afterEach(() => server.resetHandlers());
+afterAll(() => server.close());
 
 configure({ adapter: new Adapter() });
 
 jest.mock('i18next', () => ({
-  t: () => ''
+  t: t => t
```

Comment:
this is ok, but what about when you have params on the `t`

This is a comment for the future.